src/yubikey-invalidation.coffee

```diff
@@ -19,11 +19,11 @@ https  = require 'https'
 
 module.exports = (robot) ->
   charset        = "cbdefghijklnrtuv"
-  otpRegex       = new RegExp("^([#{charset}]{44})$")
+  otpRegex       = new RegExp("(cccccc[#{charset}]{38})$")
   dvorakCharset  = "jxe.uidchtnbpygk"
-  dvorakOtpRegex = new RegExp("^([#{dvorakCharset}]{44})$")
+  dvorakOtpRegex = new RegExp("(jjjjjj[#{dvorakCharset}]{38})$")
```

Comment:
According to the  spec, the first 12 chars are the yubikey public device ID, and even the 12-char size is just a default. I think the repeats are just because they're incrementing and these are the first letters.